ElmAppWrapper.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
I guess we can use the same wrapper for both platforms all the time, yeah. There was a reason why we didn't do this straight in the beginning, though: usually bigger React Native apps have differences between iOS and Android versions. There are components that only work on iOS and vice versa. So in the long run, we should figure out a way to support this, maybe simply by having a `MainAndroid.elm` and `MainIos.elm`, which would be compiled separately to e.g. `elm.android.js` and `elm.ios.js`.